extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -172,22 +172,32 @@ private function handleXmlParseError( $errors, $xmlMarkup ) {
 		return [ $renderedValue, 'markerType' => 'nowiki' ];
 	}
 
-	private function getThemeWithDefault( $params, PPFrame $frame ) {
-		$value = isset( $params[ 'theme-source' ] ) ? $frame->getArgument( $params[ 'theme-source' ] ) : false;
-		$themeName = $this->getThemeName( $params, $value );
+	private function getThemes( $params, PPFrame $frame ) {
+		$themes = [ ];
 
-		//make sure no whitespaces, prevents side effects
-		return Sanitizer::escapeClass( self::INFOBOX_THEME_PREFIX . preg_replace( '|\s+|s', '-', $themeName ) );
-	}
+		if ( isset( $params['theme'] ) ) {
+			$staticTheme = trim( $params['theme'] );
+			if ( !empty ( $staticTheme ) ) {
+				$themes[] = $staticTheme;
+			}
+		}
+		if ( !empty( $params['theme-source'] ) ) {
+			$variableTheme = trim( $frame->getArgument( $params['theme-source'] ) );
+			if ( !empty( $variableTheme ) ) {
+				$themes[] = $variableTheme;
+			}
+		}
+
+		// use default global theme if not present
+		$themes = !empty( $themes ) ? $themes : [ self::DEFAULT_THEME_NAME ];
 
-	private function getThemeName( $params, $value ) {
-		return !empty( $value ) ? $value :
-			// default logic
-			( isset( $params[ 'theme' ] ) ? $params[ 'theme' ] : self::DEFAULT_THEME_NAME );
+		return array_map( function ( $name ) {
+			return Sanitizer::escapeClass( self::INFOBOX_THEME_PREFIX . preg_replace( '|\s+|s', '-', $name ) );
+		}, $themes );
 	}
 
 	private function getLayout( $params ) {
-		$layoutName = isset( $params[ 'layout' ] ) ? $params[ 'layout' ] : false;
+		$layoutName = isset( $params['layout'] ) ? $params['layout'] : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)